samples/client/petstore/swift5/alamofireLibrary/PetstoreClient/Classes/OpenAPIs/Models/Name.swift

```diff
@@ -14,11 +14,11 @@ import AnyCodable
 public struct Name: Codable, Hashable {
 
     public var name: Int
-    public var snakeCase: Int?
+    public var snakeCase: NullEncodable<Int>
     public var property: String?
     public var _123number: Int?
 
-    public init(name: Int, snakeCase: Int? = nil, property: String? = nil, _123number: Int? = nil) {
+    public init(name: Int, snakeCase: NullEncodable<Int>, property: String? = nil, _123number: Int? = nil) {
```

Comment:
@jarrodparkes this is what I was talking about above, shouldn't we keep the optional and default value information?

Reply:
Maybe something like `snakeCase: NullEncodable<Int?> = .encodeValue(nil)`

Reply:
@4brunu 

yeh, I wasn't sure if I had enough information from the mustache file to determine whether the default should be...

```swift
snakeCase: NullEncodable<Int> = .encodeNull
// or
snakeCase: NullEncodable<Int> = .encodeNothing
// or
snakeCase: NullEncodable<Int> = .encodeSome(...)
```

How would you go about this?

Reply:
> NullEncodable<Int?>

Above, the optional type in <brackets> is redundant. I was removing the `?` by design, but let me know what you think.

Reply:
I'm not sure if that information is available or not, but looking at https://github.com/OpenAPITools/openapi-generator/issues/7521 I would say that maybe it's not available.
So maybe it will be hard to support default values

Reply:
@4brunu 

As part of the `Swift5ClientCodegen` could we do something in `postProcessModels` to detect the extension and override/set the default value?

Something like this...

| Previous Default Value | New Default Value |
|-|-|
|nil|.encodeNull|
|5|.encodeValue(5)|
|{literal}|.encodeValue({literal})|

Reply:
That's a good idea! I was about to give up on the default value.

Reply:
@4brunu I'm trying to add something like _this_ to `postProcessModels`...

```java
                if (prop.name.equals("x-null-encodable")) {
                    if (null == prop.defaultValue) {
                        prop.setDefaultValue(".encodeNull");
                    } else {
                        prop.setDefaultValue(".encodeValue(" + prop.defaultValue + ")");
                    }
                }
```

But it seems like this doesn't take effect or might be ignored. Maybe `toDefaultValue` overrides it?

Reply:
Maybe, but I don't have experience with that part of the generator